src/web_interface/templates/database/database_advanced_search.html

```diff
@@ -35,18 +35,77 @@ <h2>Advanced Search</h2>
                             <td style="padding: 5px;">
                                 <label class="checkbox-inline" style="padding: 0; margin: 0 0 0 20px;">
                                     <input type="checkbox" name="only_firmwares" value="True">
-                                    only show the firmwares that include the matching file objects
+                                    only show the firmware that includes the matching file objects
```

Comment:
better: 'show firmware including matching file objects only'

Reply:
or 'show parent firmware instead of matching file'?